### ISSUE TITLE
Fix issue in for linux-headers-5.15.0-1023-azure in index.js

### ISSUE THREAD

**Issue by bradmccoydev** [sustainable-computing-io/kepler-action#38]
There is an issue in the tests of this repo, and also the helm-chart repo as we are getting the following 404 error for http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 linux-azure-headers-5.15.0-1023 all 5.15.0-1023.29

```
Run ./
Start to build env
Linux header
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  linux-azure-headers-5.[1](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:1)5.0-1023
The following NEW packages will be installed:
  linux-azure-headers-5.15.0-1023 linux-headers-5.15.0-1023-azure
0 upgraded, 2 newly installed, 0 to remove and 9 not upgraded.
Need to get 15.0 MB of archives.
After this operation, 101 MB of additional disk space will be used.
Ign:1 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd6[4](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:5) linux-azure-headers-[5](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:6).15.0-1023 all 5.15.0-1023.29
Ign:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd[6](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:7)4 linux-headers-5.15.0-1023-azure amd64 5.15.0-1023.29
Err:1 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 linux-azure-headers-5.15.0-1023 all 5.15.0-1023.29
  404  Not Found [IP: 40.[8](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:9)1.13.82 80]
Err:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 linux-headers-5.15.0-1023-azure amd64 5.15.0-1023.2[9](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:10)
  404  Not Found [IP: 40.81.13.82 80]
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/l/linux-azure/linux-azure-headers-5.15.0-[10](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:11)23_5.15.0-1023.29_all.deb  404  Not Found [IP: 40.81.[13](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:14).82 80]
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/l/linux-azure/linux-headers-5.[15](https://github.com/sustainable-computing-io/kepler-action/actions/runs/5028930136/jobs/9020108001#step:3:16).0-1023-azure_5.15.0-1023.29_amd64.deb  404  Not Found [IP: 40.81.13.82 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
fail to install linux headers
```

**Comment by SamYuan1990** [comment]
@rootfs , we need header with ebpf right? hence for github agent, we have to update and install linux header. Is there any other options?

**Comment by rootfs** [comment]
saw this issue, it is something flaky.

**Comment by SamYuan1990** [comment]
close as fixed by #40
